br/pkg/task/restore.go

```diff
@@ -904,6 +919,11 @@ func runSnapshotRestore(c context.Context, mgr *conn.Mgr, g glue.Glue, cmdName s
 		return errors.Trace(err)
 	}
 
+	if cfg.LoadSysTablePhysical && !cfg.WithSysTable {
```

Comment:
Can we configure only one loadStatsPhysical and associate it using withSysTable?